payment/service.go

```diff
@@ -352,12 +347,17 @@ func (s *Service) CreateTransactionFromRequest(req CreateTransactionRequest, ord
 
 // CreateAnonCardTransaction takes a signed transaction and executes it on behalf of an anon card
 func (s *Service) CreateAnonCardTransaction(ctx context.Context, walletID uuid.UUID, transaction string, orderID uuid.UUID) (*Transaction, error) {
-	txInfo, err := s.wallet.SubmitAnonCardTransaction(ctx, walletID, transaction)
+	txInfo, err := s.wallet.SubmitAnonCardTransaction(
+		ctx,
+		walletID,
+		transaction,
+		"",
```

Comment:
This needs to be fixed...

Reply:
this was me early in development because i did not find where the address should sit and was trying to move forward. updated in e2068cc will add appropriate tests as well